pkg/engine/generation.go

```diff
@@ -31,16 +30,8 @@ func (e *engine) filterGenerateRules(
 	name := newResource.GetName()
 	namespace := newResource.GetNamespace()
 	apiVersion := newResource.GetAPIVersion()
-	pNamespace, pName, err := cache.SplitMetaNamespaceKey(policyNameKey)
-	if err != nil {
-		logging.Error(err, "failed to spilt name and namespace", policyNameKey)
-	}
 	resp := &engineapi.EngineResponse{
```

Comment:
We don't need policy info in the response anymore? Is it not used anywhere?

Reply:
We have the policy itself.